tools/sklearn/search_model_validation.xml

```diff
@@ -146,20 +145,20 @@ with open(outfile_estimator, "wb") as output_handler:
     </inputs>
     <outputs>
         <data format="tabular" name="outfile_result"/>
-        <data format="zip" name="outfile_estimator">
+        <data format="h5" name="outfile_estimator">
             <filter>save</filter>
         </data>
     </outputs>
     <tests>
         <test>
             <param name="selected_search_scheme" value="GridSearchCV"/>
-            <param name="infile_pipeline" value="pipeline01" ftype="zip"/>
+            <param name="infile_pipeline" value="pipeline01" ftype="h5"/>
             <conditional name="search_param_selector">
                 <param name="search_p" value="C: [1, 10, 100, 1000]"/>
                 <param name="selected_param_type" value="final_estimator_p"/>
             </conditional>
             <conditional name="search_param_selector">
-                <param name="search_p" value="k: [-1, 3, 5, 7, 9]"/>
+                <param name="search_p" value="k: [1, 3, 5, 7, 9]"/>
```

Comment:
Don't worry about the `-1`.  It was there to test the `error_score` setting on False, i.e., error tolerant.

Reply:
But, the test case would fail with this value. The number of features cannot be negative.

Reply:
Have you tried this test case? I don't think it could fail, with `error_score` set to `False`.